docs/source/tutorial/project_manager.rst

```diff
@@ -69,11 +72,11 @@ out and the model can be ran:
        'monopile_design': {},
 
        'design_phases': ['MonopileDesign'],
-       'install_phases': []
+       'install_phases': ['MonopileInstallation']
    }
 
    project = ProjectManager(config)
-   project.run_project()
+   project.run()
```

Comment:
Maybe show how to define a weathered/unweathered simulation here?

Reply:
Added a note below this section